api/resources/translations/messages-es.properties

```diff
@@ -298,72 +298,72 @@ admin.message.queue.scheduled = Programado
 analytics.anc.active-pregnancies = Embarazos activos
-analytics.anc.delivery-locations = Sitios de partos reportados
+analytics.anc.delivery-locations = Ubicaciones de partos reportados
 analytics.anc.high-risk = Embarazos de alto riesgo
 analytics.anc.missed-appointments = Citas perdidas recientes
-analytics.anc.missing-reports = Informes de parto faltando
-analytics.anc.monthly-births = Partos mensuales
-analytics.anc.monthly-registrations = Embarazos mensuales registrados
-analytics.anc.total-births = Partos totales
+analytics.anc.missing-reports = Informes de nacimiento faltantes
+analytics.anc.monthly-births = Nacimientos mensuales
+analytics.anc.monthly-registrations= Embarazos registrados mensuales
+analytics.anc.total-births = Total de nacimientos
 analytics.anc.upcoming-appointments = Próximas citas
-analytics.anc.upcoming-edds = Mujeres con FPP inminentes
+analytics.anc.upcoming-edds = Mujeres con FPP próximas
 analytics.anc.visits-completed = Visitas completadas durante el embarazo
-analytics.anc.visits-completed.description = Embarazos completos que han tenido...
+analytics.anc.visits-completed.description = Embarazos completados que han tenido...
 analytics.anc.visits-during = Visitas completadas hasta ahora
 analytics.anc.visits-during.description = Embarazos activos que han tenido...
-analytics.connection.error = Imposible recuperar datos en este momento
-analytics.reporting.change_time_unit = Cambiar la unidad de la hora
-analytics.reporting.clinics_size = Unidad de Informes
-analytics.reporting.date_range = Rango de la fecha
-analytics.reporting.district = Escoger distrito
-analytics.reporting.form = Escoger formulario
+analytics.connection.error = No se pudo obtener datos en este momento.
+analytics.reporting.change_time_unit = Cambiar la unidad de tiempo
+analytics.reporting.clinics_size = Unidad de informes
+analytics.reporting.date_range = Rango de fechas
+analytics.reporting.district = Seleccionar distrito
+analytics.reporting.form = Seleccionar formulario
 analytics.reporting.health_centers_size = SMO
-analytics.reporting.incomplete = Informes invalidos
+analytics.reporting.incomplete = Informes inválidos
 analytics.reporting.last_time_unit = Últimos {{quantity}} {{time_unit}}
-analytics.reporting.not_submitted = Informes que faltan
-analytics.reporting.reporting_rate = Tasa de notificación
+analytics.reporting.not_submitted = Informes faltantes
+analytics.reporting.reporting_rate = Análisis de los informes enviados
 analytics.reporting.reports = Informes
 analytics.target.add = Agregar objetivo
-analytics.target.aggregates = Objetivos agregados
-analytics.target.aggregates.disabled = Objetivos agregados están deshabilitados
-analytics.target.aggregates.error = Error obteniendo objetivos agregados
-analytics.target.aggregates.error.no.contact =
-analytics.target.aggregates.error.not.found = Error cargando objetivos agregados\: objetivo no encontrado.
+analytics.target.aggregates = Objetivos acumulados
```

Comment:
I find this difficult. We do not use this expression in Spanish.
Is the concept of multiple targets being brought together or considered as a whole?
We can also use: 
`Metas agrupadas`
`Objetivos consolidados`
`Objetivos conjuntos`
But I'm not a fan of any of this.

Reply:
I think it's considered as a whole, I'm going with `Objetivos consolidados`. I didn't use `Metas` for `Targets` because Targets have a setting inside called `goal` 😵‍💫 a goal in a target is, for example, 5 visits in a month.
I used Objetivo for Target, and Meta for Target's goal. 😵‍💫